src/box/lua/config/applier/runtime_priv.lua

```diff
@@ -0,0 +1,64 @@
+local function apply_rt_access(config_module)
+    local configdata = config_module._configdata
+    local credentials = configdata:get('credentials') or {}
+    credentials.roles = credentials.roles or {}
+    credentials.users = credentials.users or {}
+
+    local role_lua_call_priv = {}
+    local user_lua_call_priv = {}
+
+    local function process_lua_call(privs, user_or_role_name, user_or_role_data)
+        privs[user_or_role_name] = {}
+        local user_roles = user_or_role_data.roles or {}
+
+        if user_or_role_data.privileges then
```

Comment:
Still may have roles with `lua_call` inside.

Reply:
`lua_call` from `privileges` and `lua_call` we get from `roles` are separated now

Reply:
On a brief glance I found no test case for a user without `privileges`, but with a role that grants `lua_call`. It worth to add, I think.

Reply:
I see now, there is `no_exec` role. It verifies this case.

Reply:
Ah, no. Sorry.

Reply:
Test case added.